recipes/xbpch/meta.yaml

```diff
@@ -0,0 +1,50 @@
+
+{% set name = "xbpch" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "903358ec30cd6ca8c577aa0c9f2b384574f95c525452911b53614c5ae2046271" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - dask
+    - xarray
+
+test:
+  imports:
+    - xbpch
```

Comment:
I was going to mention that the `open_bpchdataset` failed :smile: You were faster!

```
Traceback (most recent call last):
  File "/opt/conda/conda-bld/xbpch_1504277037607/test_tmp/run_test.py", line 5, in <module>
    import xbpch.open_bpchdataset
ModuleNotFoundError: No module named 'xbpch.open_bpchdataset'
```

Reply:
I'm making quick fixes because I'm procrastinating moving my office. I'll need to do a new micro-release to update the tarball on pypi... but if you see anything else wrong with the recipe let me know :) 

Thanks for the help!